pkg/asset/installconfig/powervs/validation.go

```diff
@@ -48,6 +48,20 @@ func validateMachinePool(fldPath *field.Path, machinePool *types.MachinePool) fi
 	return allErrs
 }
 
+// ValidatePERAvailability ensures the target datacenter has PER enabled.
+func ValidatePERAvailability(client API, ic *types.InstallConfig) error {
+	capabilities, err := client.GetDatacenterCapabilities(context.TODO(), ic.PowerVS.Zone)
+	if err != nil {
+		return err
+	}
+	const per = "power-edge-router"
+	if !capabilities[per] {
+		return fmt.Errorf("%s is not available at: %s", per, ic.PowerVS.Zone)
+	}
```

Comment:
Could `"power-edge-router"` not show up in the set of capabilities? If so, maybe it's better to separate the error types (i.e. "capability not found" vs "capability not available")?

Reply:
Addressed in the latest force-push. PTAL.